ClosedXML/ClosedXML
# Fix for issue #1187

#### What's this PR do?
Fixes issue #1187 
#### Where should the reviewer start?
line 34 of XLCFCellIsConverter.cs
#### How should this be manually tested?
-set OS Formats and Location to French (France) and France respectively.
-using ClosedXML, create a spreadsheet that creates a conditional format using a number. 
   Specific example is: 
`ws.Cell(r,2).AddConditionalFormat().WhenEqualOrLessThan(-98.00002f).Font.SetFontColor(XLColor.LightGray);`
-observe that the conditional format is inserted with quotes and has other side effects (ref FrenchSpreadsheet.xlsx attached to issue #1187)
-apply this fix
-observe that the conditional format is now created correctly.

#### Any background context you want to provide?
We actually discovered this bug about 5 years ago. At the time, we submitted a bug report but we also downloaded the source, fixed the bug, and have been using our custom version of ClosedXML ever since. I have recently been refactoring our shared libraries to .netstandard and decided to try using the updated version of ClosedXML from NuGet. However, our testing has revealed that this bug still exists which is why I am submitting this pull request.

#### Screenshots (if appropriate)
#### Questions:
- Is there a blog post? No
- Does the knowledge base need an update? No
- Does this add new (C#) dependencies? No
